db/migrate/20240423112842_add_fields_to_katello_docker_manifest.rb

```diff
@@ -0,0 +1,8 @@
+class AddFieldsToKatelloDockerManifest < ActiveRecord::Migration[6.1]
+  def change
+    add_column :katello_docker_manifests, :annotations, :jsonb, default: {}
```

Comment:
I think it's fine to leave out indexes for now, but we might want to add them later if searching with in the annotations and labels becomes important. For now I think they'll just be used as information.